auth-split-migration.md

```diff
@@ -0,0 +1,59 @@
+# Module and Auth Split Migration
```

Comment:
One question I would like this to answer is how to sequence the change if you need a different auth than `public_v2`. Is it safe to specify the `auth` parameter with 0.22, then upgrade the exporter? Maybe this is obvious to experienced users of the exporter, but I'd rather make it explicit for the "someone else installed this 4 years ago and I need to keep it alive" target audience.

Reply:
No, this is already explicitly stated on line 3 that the config format in 0.23.0 has changed.

In <= 0.22.0 there was no top level `modules:` key, just a bare list of modules.

Reply:
I reworded line 3 slightly.